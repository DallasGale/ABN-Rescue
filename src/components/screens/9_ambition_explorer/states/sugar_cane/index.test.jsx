/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'

import Queensland from '.'

describe('<Queensland />', () => {
  it('renders without crashing', () => {
    const component = render(<Queensland />)
    expect(component).toMatchSnapshot()
  })
})
