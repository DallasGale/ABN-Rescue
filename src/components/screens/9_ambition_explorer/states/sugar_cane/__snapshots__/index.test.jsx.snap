// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Queensland /> renders without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="styled__StyledWrapper-qpmq7s-0 gXpTgK"
        style=""
      >
        <div
          class="scrollmagic-pin-spacer"
          data-scrollmagic-pin-spacer=""
          style="display: block; position: relative; box-sizing: content-box; min-width: 0px; width: auto; min-height: 0px; height: auto; padding-top: 0px; padding-bottom: 3000px;"
        >
          <div
            class="sticky"
            style="position: relative; margin: auto;"
          >
            <div
              class="layout-grid sugar-cane"
            >
              <div>
                <div>
                  <div
                    style="position: absolute; width: 100%; top: 120vh; opacity: 0;"
                  >
                    <h2
                      aria-hidden="true"
                      class="message first-line  typography__StyledMessageH2-sc-1n1kkhl-0 eJdLjf"
                      color="#090D2B"
                    >
                      QLD ALSO HAS MORE SUGAR CANE GROWERS THAN ANY OTHER STATE.
                    </h2>
                  </div>
                </div>
              </div>
            </div>
            <div
              style="position: absolute; width: 100%; height: 100vh; top: 120vh; opacity: 0;"
            >
              <div
                class="crop__StyledWrapper-hdml6j-0 ebfBLw"
              />
            </div>
          </div>
        </div>
        <h2
          class="ally__screen-reader-only-text"
        >
          QLD ALSO HAS MORE SUGAR CANE GROWERS THAN ANY OTHER STATE. SWEET WORK.
        </h2>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="styled__StyledWrapper-qpmq7s-0 gXpTgK"
      style=""
    >
      <div
        class="scrollmagic-pin-spacer"
        data-scrollmagic-pin-spacer=""
        style="display: block; position: relative; box-sizing: content-box; min-width: 0px; width: auto; min-height: 0px; height: auto; padding-top: 0px; padding-bottom: 3000px;"
      >
        <div
          class="sticky"
          style="position: relative; margin: auto;"
        >
          <div
            class="layout-grid sugar-cane"
          >
            <div>
              <div>
                <div
                  style="position: absolute; width: 100%; top: 120vh; opacity: 0;"
                >
                  <h2
                    aria-hidden="true"
                    class="message first-line  typography__StyledMessageH2-sc-1n1kkhl-0 eJdLjf"
                    color="#090D2B"
                  >
                    QLD ALSO HAS MORE SUGAR CANE GROWERS THAN ANY OTHER STATE.
                  </h2>
                </div>
              </div>
            </div>
          </div>
          <div
            style="position: absolute; width: 100%; height: 100vh; top: 120vh; opacity: 0;"
          >
            <div
              class="crop__StyledWrapper-hdml6j-0 ebfBLw"
            />
          </div>
        </div>
      </div>
      <h2
        class="ally__screen-reader-only-text"
      >
        QLD ALSO HAS MORE SUGAR CANE GROWERS THAN ANY OTHER STATE. SWEET WORK.
      </h2>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
